README.md

```diff
@@ -99,3 +99,10 @@ E.g. `#. The placeholder in this string represents the name of a parameter.`
 4. When a PO file reaches 100% translated and reviewed, a webhook pushes it back to the source repo ready to be consumed by your app. 
 
 5. Your app checks the user's locale settings (the browser settings for web apps, or the system settings for the CLI). If we support the user's preferred locale, the app will display strings in the user's language. Otherwise, it defaults to English.
+
+## Merge Pot files rake task
+
+The rake task that merges .pot files is present for the internationalisation of a module. This task uses 'msgcat', which is only natively present on OSes that are GNU based. For running this task locally on another OS you will need to download the gettext pkg and install it locally:
+https://pkgs.org/download/gettext
+
+Please note that it also is hardcoded to run against a 'locales/' directory, the default location for all modules generated .pot files.
```

Comment:
I believe this has changed now, and is using the default locales folder for the project's config.yaml.